examples/AWSDriverExample/src/main/java/software/amazon/ReadWriteSplittingSpringJdbcTemplateExample.java

```diff
@@ -0,0 +1,114 @@
+package software.amazon;
+
+import com.zaxxer.hikari.HikariDataSource;
+import software.amazon.jdbc.PropertyDefinition;
+import javax.sql.DataSource;
+import java.sql.SQLException;
+import java.util.Properties;
+import org.springframework.jdbc.core.JdbcTemplate;
+import software.amazon.jdbc.ds.AwsWrapperDataSource;
+import software.amazon.jdbc.plugin.readwritesplitting.ReadWriteSplittingPlugin;
+
+public class ReadWriteSplittingSpringJdbcTemplateExample {
+
+  private static final String DATABASE_URL = "database-url";
+  private static final String DATABASE_NAME = "database-name";
+  private static final String USERNAME = "username";
+  private static final String PASSWORD = "password";
+  private static final int QUERY_LOOP_NUM = 32;
+
+  public static void main(String[] args) throws SQLException {
+//     scenario1();
+//     scenario2();
+    scenario3();
+  }
+
+  private static void scenario1() {
+    // given readonly not set and using simple postrges data source implementation
+    // expect no reader instance load balancing
+    DataSource dataSource = getSimplePostgresDataSource();
+    JdbcTemplate jdbcTemplate = new JdbcTemplate(dataSource);
+    for (int i=0; i < QUERY_LOOP_NUM; i++) {
```

Comment:
```suggestion
    for (int i = 0; i < QUERY_LOOP_NUM; i++) {
```
Same for other places. You can also use IntelliJ to auto-reformat the entire file for convenience.

Reply:
Done